### ISSUE TITLE
Feature Type change based on Name tag

### ISSUE THREAD

**Issue by oxzi** [openstreetmap/iD#8588]
When editing a feature, a Feature Type is suggested based on the Name tag in the field input. However, when the given name matches a suggested category, there is no possibility to ignore it. This results in a changed Feature Type.

For example, I just tried editing a hairdresser which name has changed due to a new owner. Its new name is "New Look" which is unfortunately also the name of some clothing store.
Thus, if I try to set the name to "New Look", the UI suggests "New Look - Clothing Store". Now, it does not matter if I click this suggestion or ignore it, the Feature Type changes from "Hairdresser" to "New Look, Clothing Store". I am not able to alter the Name field without changing the Feature Type.

As a workaround, I changed the name tag below in the Tags lists.

For reproducibility, the hairdresser in question is the following node: https://www.openstreetmap.org/node/2314213770
Try editing it, deleting some characters from the Name field and typing "New Look" again.

Thanks for all your work and please excuse if this issue is a duplicate, which I failed to find.

**Comment by matkoniecz** [comment]
yes, if you search for brand name and it is not defined in NSI it will not be found

<img width="390" height="603" alt="Image" src="https://github.com/user-attachments/assets/0ec79a2e-eddc-4a04-8d80-cc8e43693d56" />

in such case you should search by feature type

<img width="391" height="747" alt="Image" src="https://github.com/user-attachments/assets/40df4acd-fc5e-42ed-a3ad-d31ca64e152d" />

**Comment by tyrasd** [comment]
This is working as expected: We do have dedicated presets for some common brands, but for smaller brands or independent businesses, the canonical way to map them is just by selecting the respective business type and entering the name and other details through the normal UI fields. I think the walkthrough tutorial explains this workflow quite well.
